libs/comments/website/src/lib/comment/comment.tsx

```diff
@@ -26,7 +39,9 @@ export const CommentHeader = styled('header')`
   display: grid;
   grid-template-columns: max-content 1fr;
   gap: ${({theme}) => theme.spacing(2)};
-  align-items: center;
+  padding-left: ${({theme}) => theme.spacing(1)};
```

Comment:
Why the paddings? Don't think thats a good idea as it will misalign everything with the content